### PR TITLE
Documentation update, added information about trackers

### PR DESCRIPTION
**Pull request by afilippx**
### Motivation and context

Related #3571

This PR adds information about trackers to the AI tools page and information about "tracker switcher" to the Objects sidebar page.
Updated the installation automatic annotation guide page.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: site/content/en/docs/manual/advanced/ai-tools.md [hunk 1]
```diff
@@ -91,3 +91,29 @@ Detectors are used to automatically annotate one frame. Supported DL models are
...
+Supported DL models are not bound to the label and can be used for any objects.
+
+- Before you start, select the `magic wand` on the controls sidebar and go to the `Trackers` tab.
+  Then select a label for the object and сlick `Track`. Then annotate the desired objects with the
+  bounding box in the first frame.
+
+  ![Start tracking an object](/images/trackers_tab.png)
+
+- All annotated objects will be automatically tracked at the moment of switching frames and if the previous frame
+  was a latest keyframe for the object. For tracking, use `F` to switch frame by frame.
```
Comment (TOsmanov) [inline site/content/en/docs/manual/advanced/ai-tools.md:107]:
```suggestion
- All annotated objects will be automatically tracked when you move to the next frame . 
  For tracking, use `Next` button on the top panel or the `F` button to move on to the next frame.
```

File: site/content/en/docs/administration/advanced/installation_automatic_annotation.md [hunk 2]
```diff
@@ -94,6 +94,8 @@ description: 'Information about the installation of components needed for semi-a
   - The number of GPU deployed functions will be limited to your GPU memory.
   - See [deploy_gpu.sh](https://github.com/openvinotoolkit/cvat/blob/develop/serverless/deploy_gpu.sh)
     script for more examples.
+  - For some models (namely [SiamMask](/docs/manual/advanced/ai-tools#trackers) you will need an [Nvidia driver](https://www.nvidia.com/en-us/drivers/unix/)
+    version greater than or equal to 450.80.02.
```
Comment (bsekachev) [inline site/content/en/docs/administration/advanced/installation_automatic_annotation.md:98]: Why do you think the version should be more or equal 450.80.02?

File: site/content/en/docs/administration/advanced/installation_automatic_annotation.md [hunk 3]
```diff
@@ -94,6 +94,8 @@ description: 'Information about the installation of components needed for semi-a
   - The number of GPU deployed functions will be limited to your GPU memory.
   - See [deploy_gpu.sh](https://github.com/openvinotoolkit/cvat/blob/develop/serverless/deploy_gpu.sh)
     script for more examples.
+  - For some models (namely [SiamMask](/docs/manual/advanced/ai-tools#trackers) you will need an [Nvidia driver](https://www.nvidia.com/en-us/drivers/unix/)
```
Comment (bsekachev) [inline site/content/en/docs/administration/advanced/installation_automatic_annotation.md:97]:
```suggestion
  - For some models (namely [SiamMask](/docs/manual/advanced/ai-tools#trackers) you need an [Nvidia driver](https://www.nvidia.com/en-us/drivers/unix/)
```

File: site/content/en/docs/manual/advanced/ai-tools.md [hunk 4]
```diff
@@ -91,3 +91,29 @@ Detectors are used to automatically annotate one frame. Supported DL models are
...
+
+- You can enable/disable tracking using `tracker switcher` on sidebar.
+
+  ![Tracker switcher](/images/tracker_switcher.jpg)
+
+- Trackable objects have indication on canvas with a model indication.
+
+  ![Tracker indication](/images/tracker_indication.png)
+
+- Tracking does not block the UI. A pop-up window displays information about the number of tracked objects.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/ai-tools.md:117]:
```suggestion
- A pop-up window displays information about the number of tracked objects.
```

File: site/content/en/docs/administration/advanced/installation_automatic_annotation.md (hunk 2 above)
Comment (TOsmanov) [reply site/content/en/docs/administration/advanced/installation_automatic_annotation.md:98]: This information is taken from https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

Comment (bsekachev) [reply site/content/en/docs/administration/advanced/installation_automatic_annotation.md:98]: Ok, I see

**Pull request merged**